csharp/README.md

```diff
@@ -0,0 +1,30 @@
+# .NET Examples
+
+![Language-Support: Stable](https://img.shields.io/badge/language--support-stable-success.svg?style=for-the-badge)
+
+This section contains all the CDK code examples written in .NET. At the moment, the CDK only supports .NET Core projects. For more information on using the CDK in .NET, please see the [Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide/work-with-cdk-csharp.html).
+
+## Running Examples
+
+To run a .NET example, execute the following:
+
+```
+$ npm install -g aws-cdk
+$ cd csharp/EXAMPLE_DIRECTORY
+$ dotnet build src
```

Comment:
You shouldn't need to explicitly build anymore.

Reply:
```suggestion
```